pandas/core/generic.py

```diff
@@ -3137,6 +3137,78 @@ def clip_lower(self, threshold, axis=None):
         subset = self.ge(threshold, axis=axis) | isnull(self)
         return self.where(subset, threshold, axis=axis)
 
```

Comment:
@jorisvandenbossche ahh, that is right. And the signature is ``.round(decimals=0, out=None)``.

ok @roblevy leave the signature alone then (and leave the ``out=`` raising)